### ISSUE TITLE
Error when property is named "parameters"

### ISSUE THREAD

**Issue by rrberry** [IBM/openapi-validator#22]
One of the models in my API definition has a property named `parameters`:
```
        "parameters": {
          "type": "object",
          "description": "A map of key/value pairs to be provided to the action.",
          "additionalProperties": {}
        }
```
The validator gives me these errors:
```
  Message :   Parameter objects must have a `description` field.
  Path    :   definitions.DialogNodeAction.properties.parameters.additionalProperties
  Line    :   11254

  Message :   Parameter type+format is not well-defined.
  Path    :   definitions.DialogNodeAction.properties.parameters.additionalProperties
  Line    :   11254
```
The errors go away if I change the name of the property to something other than `parameters`. It seems to be assuming that anything named `parameters` is an OpenAPI `parameters` object, when in this case it is just an ordinary property that happens to be named `parameters`.

**Comment by s-nilsson** [comment]
We have the same issue and turned off 2 checks in .validaterc as a workaround
```
  "oas3": {
    "parameters": {
      "no_in_property": "off",
      "missing_schema_or_content": "off",
```

**Comment by rafalbigaj** [comment]
@SamerJaser96 is there any progress on this issue?
We face this problem as well. It's quite dangerous for us to turn off checks for all parameters.

**Comment by rafalbigaj** [comment]
I found another workaround. The validator does not report errors when "parameters" property refs to the named schema.

```
  ...
  parameters:
    $ref: '#/components/schemas/GenericErrorParameters'
```

**Comment by SamerJaser96** [comment]
Yes I am currently working on this issue, should have a solution implemented soon.

**Comment by dpopp07** [comment]
Fixed in #83
